cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -568,13 +568,21 @@ impl<I: VCodeInst> VCodeBuilder<I> {
             }
 
             if let Some((dst, src)) = insn.is_move() {
-                let src = Operand::reg_use(Self::resolve_vreg_alias_impl(vreg_aliases, src.into()));
-                let dst = Operand::reg_def(Self::resolve_vreg_alias_impl(
-                    vreg_aliases,
-                    dst.to_reg().into(),
-                ));
-                // Note that regalloc2 requires these in (src, dst) order.
-                self.vcode.is_move.insert(InsnIndex::new(i), (src, dst));
+                // As we only use real registers with reg_fixed_def/reg_fixed_use constraints, the
+                // only situation where we would find a real register here is if it came from a
+                // reg_fixed_nonallocatable constraint. Don't add the alias at this point, as that
```

Comment:
s/alias/move/ -- "aliases" are what are used to make one vreg resolve to another (so that result vregs can be rewritten to the temp vregs returned by the toplevel lowering entry point), and imply a "timeless" equivalence, while "moves" are specific copy operations that occur in program flow. (Likewise the title of this PR/commit should change)